pytket/pyproject.toml

```diff
@@ -4,3 +4,5 @@ build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
 root = ".."
+write_to = "pytket/pytket/_version.py"
```

Comment:
Please add `pytket/pytket/_version.py` to the `.gitignore` file since it's not under source control.

Reply:
Yes sure will do.